packages/xo-server/src/xapi/utils.js

```diff
@@ -212,7 +211,7 @@ export const makeEditObject = specs => {
     }
 
     forEach(spec.constraints, (constraint, constraintName) => {
-      if (!isFunction(constraint)) {
+      if (!typeof constraint === 'function') {
```

Comment:
To respect homogeneity please change this to `if (typeof constraint !== 'function') {`

Reply:
Even more than homogeneity, it's a bug and will not work, good catch!